cli/main_test.go

```diff
@@ -421,7 +421,13 @@ func readOCIConfigFile(configPath string) (oci.CompatOCISpec, error) {
 	if err := json.Unmarshal(data, &ociSpec); err != nil {
 		return oci.CompatOCISpec{}, err
 	}
-
+	if ociSpec.Process != nil {
```

Comment:
I think it would be better to add this check for Process in `containerCapabilities` function itself. If the process is nil, return empty capabilities in that function itself. You can avoid checking for nil value of Process at multiple places.

Reply:
If add the check in ContainerCapabilities, we don't know whether ociSpec.Process is nil, so we still need check outside of ContainerCapabilities.

Reply:
@amshinde what do you mean, like the following ?
```
	caps, err := containerCapabilities(ocispec)  
	if err != nil {  
		return CompatOCISpec{}, err  
	}  
	ocispec.Process.Capabilities = caps  

func containerCapabilities(s CompatOCISpec) (vc.LinuxCapabilities, error) {
	if ocispec.Process == nil {
		return vc.LinuxCapabilities{}, err
	}
	...
}
```

Reply:
I think @amshinde 's suggestion is right.

```
func containerCapabilities(s CompatOCISpec) (vc.LinuxCapabilities, error) {
	if s.Process == nil {
		return vc.LinuxCapabilities{}, nil
	}
	...
}
```

This is also necessary for avoiding `contianerCapabilities()` to panic in case `s.Process` is nil

Reply:
@keloyang Yes, basically what @WeiZhang555 pasted above.